app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(10, 3)
+    assert sum(result) == 10
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(9, 3)
+    assert result == [3, 3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result = split_integer(8, 1)
+    assert result == [8]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    result = split_integer(17, 4)
```

Comment:
Simplify this test by directly asserting `split_integer(17, 4) == [4, 4, 4, 5]` to follow the checklist recommendation.